src/test/resources/application-integration.yaml

```diff
@@ -13,3 +13,10 @@ idam:
     bulkscan:
       username: bulkscanorchestrator+systemupdate@gmail.com
       password: Password12
+
+scheduling:
+  task:
+    delete-envelopes-dlq-messages:
+      enabled: true
+      cron: "*/10 * * * * *"
```

Comment:
1s is enough

Reply:
This is to run every 10 seconds

Reply:
yea, but for integration tests we don't really need to wait for 10 seconds. 1 should be enough